src/WultraMobileToken.ts

```diff
@@ -18,77 +18,63 @@ import { WMTOperations } from './operations/WMTOperations'
-import type { WMTUserAgent } from './networking/WMTNetworking'
+import { WMTUserAgent } from './networking/WMTNetworking'
 import { WMTLogger } from './WMTLogger'
 
 /**
- * MobileToken class exposes API that enables to fetch, authorize or reject basic
- * operations created in the PowerAuth stack.
+ * MobileToken class exposes APIs that enable:
+ *  Fetching, authorizing or rejecting basic
+ *  operations created in the PowerAuth stack.
+ *  Push notifications enrollment.
+ *  Inbox message management.
  */
 export class WultraMobileToken {
 
     /** Operations manager. Use for fetching pending list, approving the operations etc. */
     operations: WMTOperations
+
     /** Push manager for registering the device to recieve PowerAuth push notification for given PowerAuth activation. */
     push: WMTPush
+
     /** Inbox manager - recieve message to communicate with the user. */
     inbox: WMTInbox
 
     /**
      * 
      * @param powerAuth PowerAuth instance. Needs to be activated when calling any method of this class - othewise error will be thrown.
-     * @param baseURL BaseURL of the server. If not provided, same URL as for PowerAuth it used.
-     * @param pushBaseURL In case that the push server is on different URL.
-     *                    This is a rare scenario which doesn't happen in regular setup.
-     *                    When not set, baseURL is used.
-     * @param inboxBaseURL In case that the inbox server is on different URL.
-     *                     This is a rare scenario which doesn't happen in regular setup.
-     *                     When not set, baseURL is used.
+     * @param acceptLanguage Optionally sets the accept language for the outgoing requests headers for `operations`, `push` and `inbox` objects.
+     *                       The default value is "en".
+     *                       The value can be further modified in the each service object individualy.
+     *                       Standard RFC "Accept-Language" https://tools.ietf.org/html/rfc7231#section-5.3.5
+     *                       Response texts are based on this setting. For example when "de" is set, server
+     *                       will return operation texts in german (if available).
+     * @param userAgent Optionally sets the User agent that will be used in a HTTP hader. 
+     *                  Note that user-agent can be overriden by request processor in each API call.
      */
-    constructor(powerAuth: PowerAuth, baseURL?: string, pushBaseURL?: string, inboxBaseURL?: string) {
+    constructor(powerAuth: PowerAuth, acceptLanguage?: string, userAgent?: WMTUserAgent | string) {
 
-        let enrollmentURL = baseURL ?? powerAuth.configuration?.baseEndpointUrl ?? ""
-        let pushURL = pushBaseURL ?? enrollmentURL
-        let inboxURL = inboxBaseURL ?? enrollmentURL
+        // Retrieve the base URL and instantiate mtoken services.
+        let baseURL = powerAuth.configuration?.baseEndpointUrl ?? ""
```

Comment:
Do we want to throw here on a non-existent url?

Reply:
Hm, good question, I guess yes - throw. This should only happen in the development setup and thus throw is better than some "silent not working"

Reply:
Added.